data/json/items/containers/military.json

```diff
@@ -227,5 +227,45 @@
     "weight": "22 g",
     "volume": "480 ml",
     "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "470 ml", "max_contains_weight": "2000 g" } ]
+  },
+  {
+    "id": "55gal_drum_rubber",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "collapsible drum" },
+    "description": "Beige collapsible drum with water valve and D-Rings.  The text on the top clearly says \"POTABLE WATER ONLY\".",
+    "weight": "20 kg",
+    "volume": "10 L",
+    "price": 39500,
+    "price_postapoc": 250,
+    "//": "todo: replace with some better rubber material",
+    "material": [ "rubber" ],
+    "symbol": "0",
+    "color": "yellow",
+    "qualities": [ [ "CONTAIN", 1 ] ],
+    "flags": [ "COLLAPSE_CONTENTS" ],
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "watertight": true,
+        "rigid": false,
+        "max_contains_volume": "200 L",
+        "max_contains_weight": "500 kg"
+      }
+    ]
+  },
+  {
+    "id": "432gal_drum_rubber",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "big collapsible drum" },
+    "description": "Black collapsible drum with valve and D-Rings.  The yellow text on the top clearly says \"FLAMMABLE\".  Empty and collapsed, so you can move it elsewhere.",
```

Comment:
```suggestion
    "description": "A black collapsible drum with a valve and D-Rings.  Yellow text on its top clearly says \"FLAMMABLE\".  Empty and collapsed, it could be moved elsewhere.",
```